### PR TITLE
"Default background colour" as an option

### PR DESCRIPTION
**Pull request by PeteBa**
Added an option to use the default grafana background rather than the green OK state colour. This allows for a more muted panel that only draws attention when a warning/critical threshold is received. This modification works regardless of whether the user has selected the light or dark theme. I also tweaked the html of the Editor tab so that it has the "flip" and checkboxes inline so the tab doesn't get too deep.

You may want to check editor.html as I noticed two labels in the "Make panel gray when no data" switch around line 36 of the original file. One references isGrayColor and the other isGrayOnNoData. Is that intended?

I have used this panel a lot in my dashboards and have further customised to support Date thresholds as well as the standard Decimal, Units and Time/Date Formatting. Let me know if interested and I'll create another pull request for those additions.

Thanks for a great panel.

### CODE REVIEW

File: src/status_ctrl.js [hunk 1]
```diff
@@ -306,7 +306,8 @@ export class StatusPluginCtrl extends MetricsPanelCtrl {
 		} else if((this.series == undefined || this.series.length == 0) && this.panel.isGrayOnNoData) {
 			this.$panelContainer.addClass('no-data-state');
 		} else {
-			this.$panelContainer.addClass('ok-state');
+		    if (!this.panel.useDefaultBackground)
+		        this.$panelContainer.addClass('ok-state');
```
Comment (TomerAmirV) [inline src/status_ctrl.js]: This causes a bug with the Flip feature.
if the flip feature is on and your feature is on, the panel appears empty.
Please fix it. 😄

**Comment by TomerAmirV** [comment]
BTW,
Thanks for reporting the but with `no-data`. we fixed and merged it here: #52 

Also, feel free to add the functionality you wanted with the formatting, but we didn't quite understand what it will do... will it just add `m/s` if I choose `meter/second`?

Thanks!

**Comment by PeteBa** [comment]
Thanks for the feedback. You guys are coding fast and hard to keep in sync 8) I have changed the CSS approach to the Default Background option to support the new Flip capability.

I will take a look at porting the formatting options over the week-end. Yes, the formatting options allow you to specify number of decimal places and the units formatting will append units of measure (e.g. "m/s"). Works the same way as the format options for the standard Single Stat panel.

Let me know if any problems with this PR.

**Comment by TomerAmirV** [comment]
Thanks for fixing everything 😄 

Merging, and your changes will be released with our next version

**Comment by PeteBa** [comment]
Great stuff.

Have just created a PR for the Dates and Number formatting. This is a bit more substantial but hope it is OK and inline with how you want to develop the code.

I am now adding ability to select colours for the threshold settings and also a mode where the colour is applied to the text rather than background. Just doing some testing to make sure it works as expected.

This is proving really useful on our project. Thanks!

dashboard

**Pull request merged**